react/src/components/masthead/SprkMasthead.js

```diff
@@ -252,6 +256,10 @@ SprkMasthead.propTypes = {
    * A space-separated string of classes to add to the outermost container of the component.
    */
   additionalClasses: PropTypes.string,
+  /**
+   * Assigned to the id and the `aria-controls` attributes of the corresponding toggle and toggle content. This will be assigned a uniqueID if not provided.
```

Comment:
line length

Reply:
Would clarify to say that its for the menu button and the nav instead of toggle and toggle content.